addons/uis/apps/ios.md

```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
 
 We provide a native iOS app for openHAB, compatible with iPhone and iPad.
 It uses the REST API of openHAB to render sitemaps of your openHAB installation.
-It also supports [myopenhab.org](http://www.myopenhab.org) including push notifications.
+
+**Features:**
+
+* View and control openHAB sitemaps, similar to the BasicUI
+* Connect to local openHAB instances on the local network
+* Connect to your openHAB instance remotely using myopenHAB
+* Receive push notifications on your iOS devices via myopenHAB
+* Select a default sitemap
+* Natively supports both iPhone and iPad
+
+**Screenshots:**
+
+<div class="row">
+  <div class="col s12 m5"><img src="images/ios01.png" alt="Demo Main Menu"></div>
+  <div class="col s12 m5"><img src="images/ios02.png" alt="Demo Widget Overview"></div>
+  <div class="col s12 m5"><img src="images/ios03.png" alt="Settings"></div>
```

Comment:
"m5" means, that the image will take 5/12 of the page width on a medium sized display (a normal PC display). With two images that was okay but you'd need to work with "m4" here